### ISSUE TITLE
cygwin / tar vs 7zip

### ISSUE THREAD

**Issue by HuskerSam** [perak/kitchen-site#331]
I was having trouble running the meteor projects from meteor kitchen.  While this isn't likely to be meteor kitchen specific, it was affecting the perak:user-roles package.

I found that tar.exe was in my path (via a cygwin install).  Once I removed this from my path ... my stuff now works.  I think it found 7zip instead.

Just heads up - as I said this issue is resolved for me, but for windows users with the cygwin shell installed, tar.exe seems to work for most packages - but not all.  So if you getting an error like file not found in your appdata local temp directory - it's probably because the files are failing to unzip for whatever reason.  If you have tar or an old version of 7zip, this can very likely be the problem.

**Comment by HuskerSam** [comment]
this can be closed - this was for information purposes for windows developers.  Meteor Kitchen is a wonderful project - and I thank the contributors for their hard and productive work.

**Comment by perak** [comment]
@HuskerSam hum... I am thinking on how tar.exe affects "meteor user roles" package. What errors did you have when running (or generating?) the app? Maybe that affects meteor's automatic package install (but in that case, why only user roles...?

**Comment by HuskerSam** [comment]
While I didn't run it down more, I think having a path that starts with '/dir/dir2' - some unzippers translate to into c:\ on windows, some don't.  What I did was wipe everything I had, then the meteor update/install process would run, and it couldn't find files in my appdata/.../... folder related to the userroles package.  It probably effects more packages, but since I"m using meteor kitchen for this project it's the first one to cause a fail.  The files simply weren't there - but the system had just tried to put them there.  I found other threads about old copies of 7zip and also tar.  So if node at a low level uses tar first to unzip, then reverts to 7zip, this would seem to be the underlying cause.  Removing cygwin (and tar) from the path caused the decompression process to work.  Now that's working I haven't investigated more, just wanted to give a heads up.  I think a similar issue is effecting the 1.4.1 meteor tools download/unzip.  While I'm iffy on if it's the unix root path ('/') conversion to c:\ (which is a good guess now days, but not always in the past), this would explain my issues.  Also unix compatible tar.exe wouldn't have a clue about windows c:\.

I hope that helps, this was posted to help others that had failing meteor installs.  thanks
